### ISSUE TITLE
Link de criar conta na tela de login

### ISSUE THREAD

**Issue by jctvEC** [Openredu/Openredu#238]
Uma opção de _criar conta_ se encontra logo abaixo em letra pequena e com um tom de azul próximo do azul de fundo, o que faz com que essa opção possa passar despercebida.

![image](https://user-images.githubusercontent.com/26723995/53380170-353c8580-394b-11e9-93ca-cf8d4073fb19.png)

**Comment by fernandaramos84** [comment]
Concordo muito com esta sugestão. Acho que deveria ser priorizada pois pode aumentar o uso da ferramenta.
